@xen-orchestra/fs/src/abstract.js

```diff
@@ -435,6 +440,10 @@ export default class RemoteHandlerAbstract {
     return {}
   }
 
+  async _lock(path: string): Promise<Function> {
+    return () => {}
```

Comment:
```js
const asyncNoop = () => Promise.resolve()
```
```suggestion
    return asyncNoop
```

Reply:
Why?

Reply:
Because `proper-lockfile` returns an async function, IMHO, it makes sense to show this in the default implementation :slightly_smiling_face:

Reply:
It's the same thing just a different syntax, we already know that this implementation returns a promise which resolves in a function.

Reply:
No: `(path: string) => (() => Promise<void>)` vs `(path: string) => (() => void)`

Reply:
Ah yes you're right :+1:  the `release` is an async function